Jetchat/app/build.gradle

```diff
@@ -49,7 +49,6 @@ android {
 
     kotlinOptions {
         jvmTarget = '1.8'
-        allWarningsAsErrors = true
```

Comment:
Removing this because it's redundant. We control it from the main build.gradle file.